Classes/ViewHelper/Countdown.php

```diff
@@ -137,6 +123,13 @@ protected function getAsSeconds(int $seconds): string
      */
     protected function getFormattedMessage(int $countdownValue, string $countdownText): string
     {
-        return \sprintf($this->translator->translate('message_countdown'), $countdownValue, $countdownText);
+        return \sprintf(
+            LocalizationUtility::translate(
```

Comment:
Let let's put this `translate` call into a single line to make this method less long (vertically).